### PR TITLE
Feature/rmmis 6438 update accordion component

### PR DESCRIPTION
**Pull request by kjschulz**

### CODE REVIEW

File: src/Accordion.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```
Comment (mallenscdhhs) [inline src/Accordion.jsx]: add `'use-strict';` directive

File: src/Accordion.jsx [hunk 2]
```diff
@@ -38,14 +29,8 @@ class Accordion extends React.Component {
...
         defaultActiveKey={this.props.defaultActiveKey}
-        onSelect={this.handleSelection} >
-          {React.Children.map(this.props.children, (child, idx) => {
-            let props = Immutable.Map(child.props);
-            let cloneProps = props
-              .set('eventKey', idx)
-              .set('bsStyle', this.getActiveStyle(idx, this.state.defaultActiveKey)).toJSON();
-            return React.cloneElement(child, cloneProps, child.props.children);
-          })}
+        onClick={this.handleClick} >
```
Comment (mallenscdhhs) [inline src/Accordion.jsx]: extra little whitespace char

File: src/AccordionPanel.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (mallenscdhhs) [inline src/AccordionPanel.jsx]: add `'use-strict';` directive

File: src/AccordionPanel.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  handleClick(e) {
+    e.component = this.props;
+  }
+
+  render() {
+    return (
+      <Panel
+        {...this.props}
+        onClick={this.handleClick}
+        bsStyle={this.props.expanded ? 'info' : 'default'} >
```
Comment (mallenscdhhs) [inline src/AccordionPanel.jsx]: another little extra whitespace

File: src/index.js [hunk 5]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
 	accordion: require('./Accordion'),
+  accordionpanel: require('./AccordionPanel'),
```
Comment (mallenscdhhs) [inline src/index.js]: weird whitespace happening here

File: src/Accordion.jsx (hunk 1 above)
Comment (kjschulz) [inline src/Accordion.jsx]: Updated.

File: src/AccordionPanel.jsx (hunk 3 above)
Comment (kjschulz) [inline src/AccordionPanel.jsx]: Updated.

**Comment by mallenscdhhs** [comment]
:100: :+1:

**Pull request merged**